cogs/Voting.py

```diff
@@ -693,21 +700,21 @@ def create_vote(self, author_id, guild_id, title):
         :param title: title of the vote
         :return: the newly created Vote object
         """
-        v_id = self.gen_vote_id()
-        vote = Vote(v_id, title, author_id, guild_id, self.DBManager)
-        self.vote_lookup[(author_id, title)] = v_id
+        vote_id = self.gen_vote_id()
```

Comment:
doesn't convert these fields to strings. They'll be added to the table and converted probably, and when theyre taken out again they probably will be strings, but wouldn't they need to be converted here for the in-memory votes?

Reply:
actually, the tests should catch if it doesnt work as it uses in-memory and db-loaded votes